components/automate-gateway/gateway/middleware/auth_interceptor.go

```diff
@@ -98,8 +99,9 @@ type AnnotatedAuthorizationResponse interface {
 }
 
 type authInterceptor struct {
-	authn authn.AuthenticationClient
-	authz GRPCAuthorizationHandler
+	authn  authn.AuthenticationClient
+	authz  GRPCAuthorizationHandler
+	policy v2.PoliciesClient
```

Comment:
I think this change is no longer applicable...?

Reply:
ha i deleted this like 5 times 😅